src/client/components/slider/slider.js

```diff
@@ -2,23 +2,33 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Slider from 'react-rangeslider';
 import 'react-rangeslider/lib/index.css';
+import { connect } from 'react-redux';
 
 import './slider.css';
 
 class OneSlider extends Component {
   constructor(props, context) {
     super(props, context);
+    const { effectValues } = props;
+    const {
+      value, maxValue, minValue, step,
```

Comment:
You ca destruct this directly from props

Reply:
fixed